packages/asc-ui/src/components/Alert/AlertStyle.ts

```diff
@@ -91,5 +92,9 @@ export default styled.div<Props>`
           color: ${themeColor('tint', 'level1')};
         }
       `}
+      ${level === 'warning' &&
+      css`
+        border: 2px solid ${themeColor('secondary')};
```

Comment:
Does this border change the size of the alert itself? Is that something we should be concerned about?

Reply:
It does actually. I'll fix it

Reply:
Fixed with `box-shadow`